### PR TITLE
Fig b1

### PR DESCRIPTION
**Pull request by stephendl102**
calculateIsobologram returns different values either because of a difference in polyfc() or due to using +=. Is += supported in Julia?

### CODE REVIEW

**Comment by aarmey** [comment]
Julia has `+=`. Build is failing?

**Comment by stephendl102** [comment]
@aarmey I don't know why the build is failing. This is the error message I see on jenkins: "ERROR: Package FcgR errored during testing"

**Comment by aarmey** [comment]
There's a random, occasional convergence error, but ignore that. Here's the real one: `ERROR: BoundsError: attempt to access 16-element Array{Float64,1} at index [20]`

File: src/figures/figureB1.jl [hunk 1]
```diff
@@ -52,18 +52,18 @@ function PlotSynGraph()
...
         A = synergyGrid(4, value, FcgR, Kav)
         h = collect(Iterators.flatten(A))
-        S[ii, 1:4] = h[1:4]
-        S[ii, 5:7] = h[6:8]
-        S[ii, 8:9] = h[11:12]
-        S[ii, 10] = h[16]
+        S[ii, 1:5] = h[2:6]
+        S[ii, 5:8] = h[8:11]
+        S[ii, 8:9] = h[14:15]
+        S[ii, 10] = h[20]
```
Comment (aarmey) [inline src/figures/figureB1.jl]: So here.

Comment (stephendl102) [reply src/figures/figureB1.jl]: Ok, I resolved this issue. It still won't build though.

**Comment by stephendl102** [comment]
@aarmey It says that there is 1 approval, but that merging is blocked

**Comment by aarmey** [comment]
Huh... I'll just merge.

**Pull request merged**